tests/conftest.py

```diff
@@ -37,21 +35,6 @@ def unused_port() -> int:
     return port
 
 
-@pytest.fixture(scope='session')
-def loop() -> Iterator[asyncio.AbstractEventLoop]:
-    loop = asyncio.new_event_loop()
```

Comment:
FYI removing the per-test loop may mean that tests mutating it may influence each other depending on a shared state.